CHANGELOG.rst

```diff
@@ -1,3 +1,13 @@
+**0.7.0 - 09/08/23**
+ - Added generate_taxes_1040 method
```

Comment:
```suggestion
 - Added generate_taxes_1040 function
```
I think this is a function not a method because it's not associated with an object? Also the page in the API reference is called "Dataset Generation Functions".